code/game/objects/items/RCD.dm

```diff
@@ -276,6 +402,123 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 	. = ..()
 	AddElement(/datum/element/openspace_item_click_handler)
 
+	//Create the radial menu as a static list only at the beginning of the map for all RCD instances to share
+	if(!mapload)
+		return
+
+	var/list/icon_list = list()
+	var/list/essentials = list(
+		'icons/hud/radial.dmi' = list("wallfloor", "windowsize", "windowtype", "grillewindow", "cnorth", "csouth", "ceast", "cwest", "chair", "stool", "windoor", "secure_windoor", "airlock", "airlocktype", "access", "computer_dir"),
```

Comment:
This all looks like a nightmare to maintain, please make a unit test that ensures all these exist

Reply:
I have no idea how to write unit tests. Also why should i be the only one to do that? The people who wrote the radial menu before me didn't write these checks ,I am using the exact icons as the previous radial menu you can check the video above.
Also this is radial menu stuff so im still waiting on your word to remove this.